python_course_1604/class_01/tests/test_04.py

```diff
@@ -12,6 +14,9 @@
 
 
 @skip_if_exercize_not_started(exercize)
+@unittest.skipIf(
```

Comment:
This is what I thought originally. What was the thinking behind the two skipIfs?
/GSt

From: Federico Ressi [mailto:notifications@github.com]
Sent: Wednesday, May 4, 2016 1:38 PM
To: FedericoRessi/pythoncourse <pythoncourse@noreply.github.com>
Cc: Stedje, Gustav <gustav.stedje@intel.com>; Comment <comment@noreply.github.com>
Subject: Re: [FedericoRessi/pythoncourse] Gstedje (#20)


In python_course_1604/class_01/tests/test_04.py<https://github.com/FedericoRessi/pythoncourse/pull/20#discussion_r62030902>:

> @@ -12,6 +14,9 @@

>

>

>  @skip_if_exercize_not_started(exercize)

> +@unittest.skipIf<mailto:+@unittest.skipIf>(

E TypeError: skipIf() missing 1 required positional argument: 'reason'

You need only one skipIf:

@skip_if_exercize_not_started(exercize)

@unittest.skipIf(socket.gethostname() == 'omalleypark-1')

class TestStackLimit(unittest.TestCase):

    ...

—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/FedericoRessi/pythoncourse/pull/20/files/1809ef13da2e1038db304b564d984b4dfc66fa24#r62030902>
--------------------------------------------------------------
Intel Research and Development Ireland Limited
Registered in Ireland
Registered Office: Collinstown Industrial Park, Leixlip, County Kildare
Registered Number: 308263


This e-mail and any attachments may contain confidential material for the sole
use of the intended recipient(s). Any review or distribution by others is
strictly prohibited. If you are not the intended recipient, please contact the
sender and delete all copies.